bc_obps/registration/schema/v1/user_operator.py

```diff
@@ -151,7 +151,7 @@ class UserOperatorContactIn(ModelSchema):
 
     class Meta:
         model = Contact
-        exclude = ["id", "documents", "business_role", "address", "email", "phone_number", *AUDIT_FIELDS]
+        exclude = ["id", "business_role", "address", "email", "phone_number", *AUDIT_FIELDS]
```

Comment:
I know this is a change to a v1 file, but as documents are excluded, I don't foresee any problems